dactyl-config.yml

```
@@ -1123,7 +1135,7 @@ pages:
         nav_omit: true
         targets:
             - en
-            - ja
+            - ja          
```

Please don't add trailing whitespace:

```suggestion
            - ja
```